packages/purpose-process/test/activatePurposeVersion.test.ts

```diff
@@ -321,7 +321,7 @@ describe("activatePurposeVersion", () => {
     const expectedPurpose: Purpose = {
       ...mockPurpose,
       versions: [purposeVersionMock, purposeVersion],
```

Comment:
The PR is good but I would do another change here, I think it would be better to craft `purposeVersion` manually, in order to test it properly because now we are using the version returned from `activatePurposeVersion`, if there's a bug in that function we can't detect it with this test, since we are using the "bugged" version.

Let me know if this makes sense to you